components/webfield/ProgramChairConsole/ReviewerStatus.js

```diff
@@ -177,6 +180,21 @@ const ReviewerStatus = ({ rowData }) => {
           )
         })}
       </div>
+
+      {registrationNotes?.length > 0 && (
```

Comment:
We could move it over, I don't have a strong opinion. I chose the Status column because it was larger, so there was more more room for the note contents. That could be tweaked though.